master/usage/ipv6.md

```diff
@@ -1,33 +1,147 @@
 ---
-title: IPv6 Support
+title: Enabling IPv6 Support
 canonical_url: 'https://docs.projectcalico.org/v3.0/usage/ipv6'
 ---
 
-{{site.prodname}} supports connectivity over IPv6, between compute hosts, and
-between compute hosts and their VMs. This means that, subject to
-security configuration, a VM can initiate an IPv6 connection to another
-VM, or to an IPv6 destination outside the data center; and that a VM can
-terminate an IPv6 connection from outside.
-
-## Requirements for containers
-
-Containers have no specific requirements for utilizing IPv6
-connectivity.
-
-## Requirements for guest VM images
-
-When using {{site.prodname}} with a VM platform (e.g. OpenStack), obtaining IPv6
+### About enabling IPv6
+
+After enabling IPv6:
+- Workloads can communicate over IPv6.
+- Workloads can initiate connections to IPv6 services.
+- Workloads can terminate incoming IPv6 connections.
+
+Support for IPv6 and the procedure for enabling it varies by orchestrator.
+Refer to the section that corresponds to your orchestrator for details.
+
+- [Enabling IPv6 with Kubernetes](#enabling-ipv6-with-kubernetes)
+- [Enabling IPv6 with OpenStack](#enabling-ipv6-with-openstack)
+
+## Enabling IPv6 with Kubernetes
+
+### Limitations
+
+- Currently Kubernetes supports only one IP stack version at a time. This
+  means that if you configure Kubernetes for IPv6 then {{site.prodname}}
+  should be configured to assign only IPv6 addresses.
+- The steps and setup here have not been tested against an existing IPv4
+  cluster and are intended only for new clusters.
+
+### Prerequisites
+
+#### Host prerequisites
+
+- Each Kubernetes host must have an IPv6 address that is reachable from
+  the other hosts as well as an IPv4 address.
+- Each host must have the sysctl setting `net.ipv6.conf.all.forwarding`
+  setting it to `1`.  This ensures both Kubernetes service traffic
+  and Calico traffic is forwarded appropriately.
+- Each host must have a default IPv6 route.
+
+#### Kubernetes components prerequisites
+
+Kubernetes components must be configured to operate with IPv6.
+To enable IPv6, set the following flags.
+- For kube-apiserver:
+  - The `--bind-address` or `--insecure-bind-address` flags should be set
+    to the appropriate IPv6 address or `::` for all IPv6 addresses on
+	the host.
```

Comment:
formatting nit... but a lot of these end with a couple of words on a new line when I don't think they need to. Why not just have "the host" on the line above it?